apps/innovations/_services/innovations.service.ts

```diff
@@ -69,6 +69,8 @@ import { groupBy, isString, mapValues, omit, pick, snakeCase } from 'lodash';
 import { BaseService } from './base.service';
 import type { InnovationDocumentService } from './innovation-document.service';
 import SYMBOLS from './symbols';
+import { InnovationRelevantOrganisationsStatusView } from '@innovations/shared/entities/views/innovation_relevant_organisations_status_view.entity';
```

Comment:
`@innovations/shared/entities`

Reply:
How do I change this on?
![image](https://github.com/user-attachments/assets/c49d8157-3885-467f-bc51-77be8a98640d)
In the same file we also have:
`import {
  InnovationListViewWithoutNull,
  InnovationProgressView
} from '@innovations/shared/entities/views/innovation-progress.view.entity';`